Common/tests/Auth/AuthenticationIntegrationTest.cs

```diff
@@ -1182,6 +1182,9 @@ public static IEnumerable GetCases(List<CasesConfig> casesConfigs)
                     false,
                     true,
                   })]
+  [SuppressMessage("Style",
+                   "IDE0060:Remove unused parameter",
+                   Justification = "Required for TestCaseSource")]
```

Comment:
I think it would be better to have `_ = unusedParameter;` instead of this annotation.
By the way, This does not compile.
> /home/runner/work/ArmoniK.Core/ArmoniK.Core/Common/tests/Auth/AuthenticationIntegrationTest.cs(1185,4): error CS0246: The type or namespace name 'SuppressMessageAttribute' could not be found (are you missing a using directive or an assembly reference?) [/home/runner/work/ArmoniK.Core/ArmoniK.Core/Common/tests/ArmoniK.Core.Common.Tests.csproj]

Reply:
Now it does...
Your solution wouldn't suppress the compiler message.

The proper solution would be to have test that actually use these parameters.
Also, in the current implementation, tests cannot be launched with the debugger...

Reply:
I think the output is actually used to "transmit" the type from Nunit to the actual function.

Reply:
In this case, yes. But in the case of `AuthServiceShouldGiveUserInfo` (line 1316 of same file), no...
IMHO, not being able to launch the tests easily nor to debug them is an issue...
I'd rather have C# code generated but directly accessible than reflexion.